modules/core/24-host/v2/packet_keys.go

```diff
@@ -2,24 +2,22 @@ package v2
 
 import (
 	"fmt"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
 // PacketReceiptKey returns the store key of under which a packet
 // receipt is stored
 func PacketReceiptKey(channelID string, sequence uint64) []byte {
-	return []byte(fmt.Sprintf("receipts/channels/%s/sequences/%s", channelID, sdk.Uint64ToBigEndian(sequence)))
+	return []byte(fmt.Sprintf("receipts/channels/%s/sequences/%d", channelID, sequence))
```

Comment:
wdym by corrupted? String value for key is only intermediate repr before casting to bytes, no? Is there missing context whereby we care about string value for these?

Looking at spec, these will continue using BigEndian encoding for seqs to keep sorted order.

Reply:
This is what the string ends up looking like: `"acks/channels/channel-0/sequences/\x00\x00\x00\x00\x00\x00\x00\x01` 
and it creates some issues with our relayer. It should technically not be a huge problem, but I can't imagine this being the idea of what is should look like at least :)

Reply:
I think the main reason for doing this was to make it so iteration over channel IDs/ sequences etc happens in an expected order, rather than lexicographically. We are doing this also in the PacketForwardKey fn.

Reply:
yup, specifically https://github.com/cosmos/ibc-go/issues/7132

Reply:
Alright, so we want it to look like this? If so, I can go back and figure out a way to  deal with those characters in the operator cli

Reply:
yea. Can't you just decode the byte slice into the number where needed?